taskbuilder.js

```diff
@@ -129,8 +129,18 @@ function getTask(ops) {
         host,
         actions;
 
+    // alias prod(uction) to the production hostname
     if (ops.host.match(/prod(uction)?/)) {
         host = DDG;
+    // if hostname contains a dot, assume it's a full one
+    // rather than a subdomain
+    } else if (ops.host.indexOf(".") > -1) {
+        host = ops.host;
+    // if there's a port but no dot assume a duckduckgo subdomain with a port
```

Comment:
this is fine and not worth changing, but i believe you don't need it.  the resolver on all ddg linux boxes will assume `.duckduckgo.com` if its not a fully qualified domain name.  same for the else block below.